static/js/app.js

```diff
@@ -847,6 +870,95 @@ $(function () {
             this.fetchMarkers();
             this.updateFilterString();
         },
+        loadFilterFromParameters: function() {
+            var bool_atrs = {};
+            bool_atrs["show_discussions"] = this.showDiscussion;
```

Comment:
I think object attributes in JS can be accessed as hash keys too, so you could just save the list of attributes and get each from `this`. Am I wrong?
Only for `showDiscussion` the name is different, but that should be fixed anyway for consistency.